products/eks.md

```diff
@@ -102,19 +114,23 @@ releases:
 > conformant for a predictable experience.
 
 Amazon EKS guarantees support for at least four production-ready versions of Kubernetes at any given
-time. A Kubernetes version is fully supported on EKS for 14 months after first being available on
-Amazon EKS. This is true even if upstream Kubernetes is no longer supporting a version available on
-Amazon EKS.
+time. A Kubernetes version is fully supported on EKS for 14 months as standard support and for up to
+26 months as exteneded support after first being available on Amazon EKS. This is true even if
+upstream Kubernetes is no longer supporting a version available on Amazon EKS.
 
 You can subscribe to upgrade notices (sent approximately 12 months after the Kubernetes version was
 released on Amazon EKS) on your [Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/).
 The notice includes the end of support date, which is at least 60 days from the date of the notice.
 
 ## Upgrading
 
+At the end of the of standard support for a Kubernetes version, Amazon EKS clusters running on EOL
+version will automatically enter extended support and Amazon apply an additional charge per cluster
+hour for all clusters running an extended support version. 
+
 Amazon EKS will automatically upgrade existing control planes (not nodes) to the oldest supported
-version through a gradual deployment process after the end of support date. After the automatic
-control plane update, you _must [manually update cluster add-ons and Amazon EC2 nodes](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html#update-existing-cluster)_.
+version through a gradual deployment process after the end of extended support date. After the
+automatic control plane update, you _must [manually update cluster add-ons and Amazon EC2 nodes](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html#update-existing-cluster)_.
```

Comment:
This is too focused on extended support. From what I understand this is true for both level of support so, I think it would be best to change `after the end of extended support date` to `after the end of support date, whether standard or extended`. WDYT ?

Reply:
I don't know, but considering this will be a paid option I would find odd that you can't opt-out of it.

> During the preview, there is no additional cost to run a cluster with extended support. When extended support is generally available in early 2024, there will be an additional charge per cluster hour for all clusters running an extended support version. Alternatively, you can upgrade the cluster at any time to a standard support version, which remain on the existing [Amazon EKS Pricing](https://aws.amazon.com/eks/pricing/).

@endoflife-date/everyone, is there anyone knowing Amazon EKS that could help here ?

Reply:
> With extended support, Amazon EKS clusters will keep running on a version for up to 12 months after the 14-month initial support period. At the end of the 14 months of standard support for a Kubernetes version, clusters running on that version will automatically enter extended support. **There’s no action or configuration changes required.** At the end of the extended support period, any clusters using the deprecated version automatically upgrade to the next-oldest version. **If you don’t want your cluster to use extended support for a Kubernetes version, you can upgrade it at any time to a version in the standard support window.**

There is opt-out by upgrading to the supported Kubernetes version :no_mouth:

Reply:
Lol ! Would be great to also add a link to https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#extended-support-faqs in the description so reader can easily confirm.